powerspikegg/computation_models/match/BUILD

```diff
@@ -41,6 +41,14 @@ py_library(
     srcs = ["train.py"],
     visibility = ["//visibility:public"],
     deps = [
-        "@pydep_tensorflow//:library",
+        ":tensorflow",
     ],
 )
+
+alias(
```

Comment:
This alias is only for the pypi package and won't work for the c++ therefore we can't replace the //third_party/tensorflow module.

However this point out that we have to make a more explicit distinction between the source submodule and the pypi package.